packages/@adobe/spectrum-css-temp/components/popover/index.css

```diff
@@ -144,3 +144,10 @@ governing permissions and limitations under the License.
     margin-left: calc(var(--spectrum-global-dimension-size-150) * -1);
   }
 }
+
+@media screen and (max-height: 400px) {
```

Comment:
where did this value come from?

Reply:
Perhaps from here: https://github.com/adobe/react-spectrum/blob/9a9327fae5b5a466f7a1394a82669ea19858356d/packages/%40adobe/spectrum-css-temp/components/dialog/index.css#L438